packages/ui/src/hooks/use-card.ts

```diff
@@ -0,0 +1,61 @@
+import { useAppearance } from '~/contexts';
+
+import { useDisplayConfig } from './use-display-config';
+
+/**
+ * Access card rendering values for the `<Card.Logo />` and `<Card.Footer />`
+ * based on the display config and parsed appearance.
+ *
+ * @example
+ * ```tsx
+ * import { useCard } from './use-card';
+ *
+ * function MyCardComponent() {
+ *   const { logoProps, footerProps } = useCard();
+ *
+ *   return (
+ *     <Card>
+ *       <Card.Logo {...logoProps} />
+ *       <Card.Footer {...footerProps} />
+ *     </Card>
+ *   );
+ * }
+ * ```
+ *
+ * @returns {{
```

Comment:
I don't think we need the `@returns` annotation here since TypeScript is already inferring a return type, right?

Reply:
👍🏼 it is yeah